### ISSUE TITLE
What datasets are used

### ISSUE THREAD

**Issue by Pink-Dolphin** [hertz-pj/SNAC-Vocos#2]
May I ask what datasets you are using and what is the appropriate setting of batch_size num_workers and val_check_interval for training? I use LibriSpeech train-clean-100 as the training set. dev-clean acts as a validation set, but the loss reduction seems to be slow。

**Comment by hertz-pj** [comment]
The datasets include LibriTTS, Hi-Fi TTS, VCTK, AISHELL-1, and AISHELL-3.
You can mainly refer to the Mel Loss curve. Perhaps you can refer to my training curve.
![企业微信截图_1733971689990](https://github.com/user-attachments/assets/ae920f87-a63c-4efe-b27e-4b3bc8a74d0e)
![image](https://github.com/user-attachments/assets/933aa341-d60a-44a4-92e8-321282c716a4)

**Comment by Pink-Dolphin** [comment]
Thank you very much for your reply. I have learned that I need to expand my data set. In addition, is your training config basically the same as the config file you submitted? Do I need to modify some parameters or configurations

**Comment by hertz-pj** [comment]
The training config is the same as the config in the repo. You don't need to modify anything.

**Comment by Pink-Dolphin** [comment]
Thank you. In addition, I am sorry that I am not particularly familiar with the calculation of compression ratio. I would like to know how much kbps will be obtained by the model processing after the parameter configuration of this setting.

**Comment by hertz-pj** [comment]
80frames/s

**Comment by Pink-Dolphin** [comment]
Thank you for sharing.
I've followed the instructions to train a custom model.  The tensorboard is showing decent progress and audio predictions are starting to sound good. I've set the resume parameter in config,But I am unable to load custom model checkpoint for inference.  Can you share how to use the custom trained checkpoints for inference?
![PixPin_2024-12-30_12-28-59](https://github.com/user-attachments/assets/86a085e0-a5b1-4b43-9df5-da11e482e4d2)

**Comment by hertz-pj** [comment]
For inference, there is no need to modify the resume parameter in the configuration file. Please refer to infer.py.
